firefox-ios/Client/Frontend/UserContent/UserScripts/AllFrames/AtDocumentStart/DownloadHelper.js

```diff
@@ -59,4 +60,9 @@ Object.defineProperty(window.__firefox__, "download", {
 });
 }
 
-
+document.addEventListener("click", (event) => {
```

Comment:
@DanielDervishi put a small comment on what it does in terms of what APP_ID_TOken is used for etc.